SofaKernel/modules/SofaCore/src/sofa/core/topology/Topology.cpp

```diff
@@ -40,4 +40,12 @@ bool Topology::removeInNode(objectmodel::BaseNode* node)
     return true;
 }
 
+// Tetrahedron
+const unsigned int edgesInTetrahedronArray[6][2] = {{0,1}, {0,2}, {0,3}, {1,2}, {1,3}, {2,3}};
```

Comment:
I did not find them. But this would be the place to define them if need be.

Reply:
they are here: https://github.com/sofa-framework/sofa/blob/0ba23dce74ddd0f2b6727048009c7db2289bb367/SofaKernel/modules/SofaCore/src/sofa/core/topology/BaseMeshTopology.h#L338

Reply:
btw we added the term "orientation" to differentiate this table from the trianglesInTetrahedron real buffer which contains the actual ids of the triangles of a given tetrahedron

Reply:
Ok, it would be great to put them all together. What do you think? In Topology.h or in BaseMeshTopology.h?
Moreover, trianglesOrientationInTetrahedronArray is statically defined. It means there are as many instances of this variable as files including BaseMeshTopology.h. I recommend using the extern keyword to declare it, and define it in a cpp file.

Reply:
Yes you can move them in Topology.h
ok as well for extern rather than static.

Reply:
I am concerned by the similarity of `quadsInHexahedronArray` and `quadsOrientationInHexahedronArray` which both represent quads in a hexa, but with a different order.